packages/engine/src/common/functions/logger.ts

```diff
@@ -95,36 +105,47 @@ const multiLogger = {
       // For non-local builds, this send() is used
       const send = (level) => {
         const url = new URL('/api/log', config.client.serverUrl)
+        const consoleMethods = {
+          debug: console.debug.bind(console, `[${opts.component}]`),
+          info: console.log.bind(console, `[${opts.component}]`),
+          warn: console.warn.bind(console, `[${opts.component}]`),
+          error: console.error.bind(console, `[${opts.component}]`),
+          fatal: console.error.bind(console, `[${opts.component}]`)
+        }
 
         return async (...args) => {
-          const consoleMethods = {
-            debug: console.debug.bind(console, `[${opts.component}]`),
-            info: console.log.bind(console, `[${opts.component}]`),
-            warn: console.warn.bind(console, `[${opts.component}]`),
-            error: console.error.bind(console, `[${opts.component}]`),
-            fatal: console.error.bind(console, `[${opts.component}]`)
-          }
-
-          // @ts-ignore
-          const logParams = encodeLogParams(...args)
-
-          // In addition to sending to logging endpoint,  output to console
-          consoleMethods[level](...args)
-
-          // Send an async rate-limited request to backend /api/log endpoint for aggregation
-          // Also suppress logger.info() levels (the equivalent to console.log())
-          if (config.client.serverHost && level !== 'info') {
-            logRequestCache.set(logParams.msg, () =>
-              fetch(url, {
-                method: 'POST',
-                headers: { 'Content-Type': 'application/json' },
-                body: JSON.stringify({
-                  level,
-                  component: opts.component,
-                  ...logParams
+          try {
+            // @ts-ignore
+            const logParams = encodeLogParams(...args)
+
+            // In addition to sending to logging endpoint,  output to console
+            consoleMethods[level](...args)
+
+            // Send an async rate-limited request to backend logs-api service for aggregation
+            // Also suppress logger.info() levels (the equivalent to console.log())
+
+            if (config.client.serverHost && level !== 'info') {
+              logRequestCache.set(logParams.msg, () =>
+                fetch(url, {
```

Comment:
`url` points to `'/api/log'` which is no longer there. So I think this caching is not working.

Reply:
Done